### DISCUSSION TITLE
Custom pivot entity standalone one-to-many field for pivot entity update

### DISCUSSION THREAD

**Discussion by QuestionAndAnswer** [mikro-orm/mikro-orm#3630] [Q&A] [answered]
In the documentation about [custom pivot entity](https://mikro-orm.io/docs/collections#custom-pivot-table-entity) we have this mentioned:

>We can as well define the 1:m properties targeting the pivot entity as in the previous example, and use that for modifying the collection, while using the M:N property for easier reading and filtering purposes.

This statement is quite ambiguous in terms of actual implementation.
Does that mean that I do have additional `productsPivots` field like this (modifying documentation's example?

```typescript
@Entity()
export class Order {
  @OneToMany(() => OrderItem, 'order')
  productsPivots = new Collection<OrderItem>(this);

  @ManyToMany({ entity: () => Product, pivotEntity: () => OrderItem })
  products = new Collection<Product>(this);
}
``` 

Is it a correct interpretation of what's been written in the documentation?
Can I modify OrderItem fields (after getting it from collection) as ordinary entity, and still have MikroOrm manage ManyToMany actual relation?

**Comment by QuestionAndAnswer** [comment, marked as answer]
Okay, seems like I found answers already:

https://github.com/mikro-orm/mikro-orm/discussions/3036#discussioncomment-2600570
https://github.com/mikro-orm/mikro-orm/discussions/3531

In short: yes it is the way to do it.

**Reply by B4nan** [reply]
Correct. What is ambiguous about this? That sentence comes at the end of the section, there is no example connected to it. Feel free to send a PR making things more understandable to you, or adding an example for this.
